lib/utils/fetcher.ts

```diff
@@ -0,0 +1,14 @@
+const BASE_BACKEND_URL = "http://abeghelp-backend-staging.up.railway.app"; //TODO - Find a way to hide this via environment variables
```

Comment:
create an env.local file by the time you finalize this PR

Reply:
Speaking of env, do you happen to know how NextJs's bundler handles emv variables? 

From what I keep hearing, most bundlers still end up shipping the env variables to the minified code in prod...👀

Reply:
I've not heard about this. I do know that the way to expose env to the client is by using the `NEXT_PUBLIC` prefix. Plus, i will still add the endpoint to vercel manually

Reply:
Fair enough then, I'll leave that worry for later it seems 👍

Reply:
Is the ENV will be used on the client Vercel encrypts it tho

Reply:
Fair enough then, will leave this worry for later👍

Reply:
> If the ENV will be used on the client Vercel encrypts it tho

Just what I needed to hear. So it's all safe then👍

On that note tho, which section of the NextJs docs could I find this information?

Reply:
> > If the ENV will be used on the client Vercel encrypts it tho
> 
> Just what I needed to hear. So it's all safe then👍
> 
> On that note tho, which section of the NextJs docs could I find this information? Can't seem to find it for some reason

Reply:
@Danieljs-codes

Reply:
Just to chime in here, the env encryption done by vercel is env's related to server actions in the app dir, so its not applicable here.

in this case it is perfectly safe to expose the url of our backend endpoints, because you cannot fully hide them anyway, they will be visible in the network tab regardless haha.

It is also best practice to put them in a public ENV variable so it is reusable app wide without copy pasting everywhere

Reply:
> Just to chime in here, the env encryption done by vercel is env's related to server actions in the app dir, so its not applicable here.
> 
> in this case it is perfectly safe to expose the url of our backend endpoints, because you cannot fully hide them anyway, they will be visible in the network tab regardless haha.
> 
> It is also best practice to put them in a public ENV variable so it is reusable app wide without copy pasting everywhere

Thanks for the clarification🤝